data/src/test/java/org/apache/iceberg/data/TestGenericReaderDeletes.java

```diff
@@ -18,23 +18,25 @@
  */
 package org.apache.iceberg.data;
 
+import static org.assertj.core.api.Assertions.assertThat;
+
 import java.io.File;
 import java.io.IOException;
+import java.nio.file.Files;
 import org.apache.iceberg.PartitionSpec;
 import org.apache.iceberg.Schema;
 import org.apache.iceberg.Table;
 import org.apache.iceberg.TestTables;
 import org.apache.iceberg.io.CloseableIterable;
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
 import org.apache.iceberg.util.StructLikeSet;
-import org.junit.Assert;
 
 public class TestGenericReaderDeletes extends DeleteReadTests {
```

Comment:
I believe this should be parameterized now (since the base class requires parameterization due to usage of `@TestTemplate`)

Reply:
maybe we should move the `parameters()` definition from `TestFlinkReaderDeletesBase` into `DeleteReadTests`

Reply:
Yeah i'll move the `parameters()` and parameterize the file